src/scripts/content/slack.js

```diff
@@ -1,12 +1,10 @@
 'use strict';
 /* global togglbutton, $ */
 
-togglbutton.render('#channel_name:not(.toggl)', { observe: true }, () => {
-  const placeholder = $('.channel_title_info');
-  const projectName = $('#team_name').textContent.trim();
-  const description = $('#channel_name')
-    .textContent.trim()
-    .replace(/^#/, '');
+togglbutton.render('[data-qa="channel_name"]:not(.toggl)', { observe: true }, () => {
```

Comment:
This is the same as someone would create a script to rely on `data-test-id` on toggl DOM elements 😁

Reply:
Well, I figured these are less likely to change than element's classes. Of course, if the interface changes regardless of the IDs things will still break, but I guess this makes us a bit less prone to errors due to small changes.